test/create.js

```diff
@@ -0,0 +1,50 @@
+var assert = require("assert");
+var Web3 = require("web3");
+var TestRPC = require("ethereumjs-testrpc");
+var Provider = require("../index");
+
+describe("Provider", function() {
+  var HttpProvider;
+
+  beforeEach("mock HttpProvider", function() {
+    HttpProvider = Web3.providers.HttpProvider;
+    Web3.providers.HttpProvider = TestRPC.provider;
```

Comment:
My gut instinct was that this monkey patch was a little gross, but then I went, "Ah, I see what you did there." I'm okay with this for now, but if you want to take the extra time and initialize an actual TestRPC server [like this](https://github.com/ethereumjs/testrpc/blob/master/test/requests.js#L930) that'd be cool too.